app/controllers/discussions_controller.rb

```diff
@@ -48,14 +51,25 @@ def update
 
   private
 
+    def check_discussion_is_upcoming
+      if !(@discussion.date.to_s > Date.current.to_s)
```

Comment:
date is in format (weekday, day month year)
to_s changes it to (year-month-day) which is why I used it....
Replace it with strftime?

On Fri, Jul 31, 2015 at 4:24 PM, Jaspreet Singh <notifications@github.com>
wrote:

> In app/controllers/discussions_controller.rb
> <https://github.com/SAkshat/iconf/pull/3#discussion_r35964077>:
>
> > @@ -48,14 +51,25 @@ def update
> >
> >    private
> >
> > +    def check_discussion_is_upcoming
> > +      if !(@discussion.date.to_s > Date.current.to_s)
>
> why to_s
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SAkshat/iconf/pull/3/files#r35964077>.
>



-- 

Akshat Singhal
Vinsol